jwst/extract_2d/extract_2d_step.py

```diff
@@ -16,7 +16,8 @@ class Extract2dStep(Step):
     spec = """
         slit_name = string(default=None)
         extract_orders = int_list(default=None)  # list of orders to extract
-        extract_height =  integer(default=None)  # extraction height in pixels
+        extract_height =  integer(default=None)  # extraction height in pixels, TSGRISM mode
```

Comment:
Since we now have 2 extraction height params, maybe we should be explicit in the names for both of them which mode they apply to, such as `tsgrism_extract_height`.